astropy/stats/tests/test_biweight.py

```diff
@@ -171,6 +193,28 @@ def test_biweight_midvariance_axis_3d():
         assert_allclose(bw[y], bwi)
 
 
+def test_biweight_scale_axis_tuple():
+    """Test a 3D array with a tuple axis keyword."""
+
+    data = np.arange(24).reshape(2, 3, 4)
+    data[0, 0] = 100.
+
+    assert_equal(biweight_scale(data, axis=0),
+                 biweight_scale(data, axis=(0,)))
+    assert_equal(biweight_scale(data, axis=-1),
+                 biweight_scale(data, axis=(2,)))
+    assert_equal(biweight_scale(data, axis=(0, 1)),
```

Comment:
This test is a bit puzzling...

Reply:
Um, yeah, that's a typo.  I intended to compare `(0, 1)` with `(1, 0)`.  Thanks for catching that!